src/databases/Uintah/avtUintahFileFormat.C

```diff
@@ -2036,8 +2040,6 @@ avtUintahFileFormat::GetVar(int timestate, int domain, const char *varname)
                           varName, matlNo, timestate,
                           plow, phigh,
                           (nodeCentered ? 0 : loadExtraGeometry));
-
-      // visitTimer->StopTimer(t2, "avtUintahFileFormat::GetVar getGridData");
```

Comment:
If you remove this commented out line, you should also remove the commented out line at 2037, where t2 is set.

Reply:
Okay fixed that, and committed to my local branch but I can not push:

% git push origin task/allen/Uintah_bugfix
To ssh://github.com/visit-dav/visit.git
 ! [rejected]              task/allen/Uintah_bugfix -> task/allen/Uintah_bugfix (non-fast-forward)
error: failed to push some refs to 'ssh://git@github.com/visit-dav/visit.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

I have done a git pull and I seem update.

What am I missing

Reply:
That should work. Can you do another "git pull" followed by "git push ..."? Anyone else have thoughts on this?

Reply:
Tried again same results. I just created a new branch and will do the pull from it.